crates/bitwarden/src/auth/tde.rs

```diff
@@ -15,7 +15,9 @@ pub(super) fn make_register_tde_keys(
     org_public_key: String,
     remember_device: bool,
 ) -> Result<RegisterTdeKeyResponse> {
-    let public_key = AsymmetricPublicCryptoKey::from_der(&STANDARD.decode(org_public_key)?)?;
+    let public_key = AsymmetricPublicCryptoKey::from_der(SensitiveVec::new(Box::new(
+        STANDARD.decode(org_public_key)?,
```

Comment:
Should we convert `org_public_key` to a sensitive value before base64 decoding it?